src/vehicle.cpp

```diff
@@ -1847,59 +1847,44 @@ int vehicle::install_part( int dx, int dy, const vpart_str_id &id, item&& obj, b
 
 int vehicle::install_part( int dx, int dy, const vehicle_part &new_part )
 {
+    // Should be checked before installing the part
+    bool enable = false;
+    if( new_part.is_engine() ) {
+        enable = true;
+    } else {
+        // @todo read toggle groups from JSON
+        static const std::vector<std::string> enable_like = {{
```

Comment:
I'm not sure about that variable naming but I don't have a better suggestion. The flags should be alphabetical to limit future merge conflicts (not a particularly serious problem here but good practice overall)